src/server/svr_func.c

```diff
@@ -1955,12 +1955,10 @@ entlim_resum(struct work_task *pwt)
 	while (pj) {
 		if ((pj->ji_qs.ji_svrflags & JOB_SVFLG_SubJob) == 0) {
 			if (is_resc) {
-				(void)set_entity_resc_sum_max(pj, pque, NULL, INCR);
-				(void)set_entity_resc_sum_queued(pj, pque, NULL, INCR);
+				account_entity_limit_usages(pj, pque, NULL, INCR, ETLIM_ACC_ALL_RES);
 			}
 			else {
-				(void)set_entity_ct_sum_max(pj, pque, INCR);
-				(void)set_entity_ct_sum_queued(pj, pque, INCR);
+				account_entity_limit_usages(pj, pque, NULL, INCR, ETLIM_ACC_ALL_CT);
```

Comment:
Here we have to `INCR` only ct limit attributes. i.e invoke only `set_entity_ct_sum_*` in `account_entity_limit_usages()`. Hence passing `ETLIM_ACC_ALL_CT` as flag.